exporter/otlp-http/README.md

```diff
@@ -0,0 +1,102 @@
+# opentelemetry-exporter-otlp
```

Comment:
`opentelemetry-exporter-otlp-http` needs its own README, or at least we need to replace all the instances of `opentelemetry-exporter-otlp` in this file.